From 98d31137456d00cb8285aa16536529a3a7a14f49 Mon Sep 17 00:00:00 2001
Message-ID: <98d31137456d00cb8285aa16536529a3a7a14f49.1747779548.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1747779547.git.sharlatanus@gmail.com>
References: <cover.1747779547.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Tue, 20 May 2025 12:11:10 +0100
Subject: [PATCH 25/31] gnu: python-roman-datamodels: Update to 0.25.0.

* gnu/packages/astronomy.scm (python-roman-datamodels): Update to 0.25.0.
[native-inputs]: Remove python-setuptools; add python-setuptools-next.

Change-Id: I7602ace125e87fd9e608ae2ee8194673948cee0f
---
 gnu/packages/astronomy.scm | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index f85e14358f..81d1c2f73c 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -7217,13 +7217,13 @@ (define-public python-regions
 (define-public python-roman-datamodels
   (package
     (name "python-roman-datamodels")
-    (version "0.24.0")
+    (version "0.25.0")
     (source
      (origin
        (method url-fetch)
        (uri (pypi-uri "roman_datamodels" version))
        (sha256
-        (base32 "1jlzfpzg13g9sq1i0dz27ymny48rwz74blp2dqhk9idynbyp5l4k"))))
+        (base32 "0sb2zpanklyh5j2rgm27lrscl51fwyny4f6c9gisj373pp7w6h2c"))))
     (build-system pyproject-build-system)
     (arguments
      (list
@@ -7240,7 +7240,7 @@ (define-public python-roman-datamodels
            python-pytest-doctestplus
            python-pytest-env
            python-pytest-xdist
-           python-setuptools
+           python-setuptools-next
            python-setuptools-scm
            python-wheel))
     (propagated-inputs
-- 
2.49.0

